.rubocop.yml

```diff
@@ -3,18 +3,10 @@ AllCops:
     - vendor/**/*
     - Guardfile
 
-AlignParameters:
```

Comment:
The various rules here are disabled to keep a standard set of rules in all cookbook projects.  It's going to aid us as we move to a custom wrapper around rubocop in the near future.  They should be added back.